meson.build

```diff
@@ -26,28 +26,28 @@ if meson.is_subproject()
   endif
 endif
 
-# Get version_commit
-version_commit = run_command(git_exe, '-C', repo, 'rev-list', '--all', '--count')
-if version_commit.returncode() != 0
-  version_commit = 0
-else
-  version_commit = version_commit.stdout().strip()
-endif
-
-# Get gittap
-gittap  = run_command(git_exe, '-C', repo, 'describe', '--tags', '--match', '[0-9]*')
-if gittap.returncode() != 0
-  gittap = ''
-else
-  gittap = gittap.stdout().strip()
-endif
-
-# Get gittip
-gittip  = run_command(git_exe, '-C', repo, 'rev-parse', 'HEAD')
-if gittip.returncode() != 0
-  gittip = 'unknown'
+if git_exe.found()
+  # Get version_commit
+  git_rev_list = run_command(git_exe, '-C', repo, 'rev-list', '--all', '--count')
+  if git_rev_list.returncode() == 0
+    version_commit = git_rev_list.stdout().strip()
```

Comment:
No default value. Will fail if there is no `.git` directory.

Reply:
fixed